readline-complete.el

```diff
@@ -274,9 +274,11 @@ To disable ac-rlc for an application, add '(prompt ac-prefix-rlc-disable).")
     (when prefix
       (funcall prefix))))
 
-;;;###autoload
+(eval-when-compile
+  (require 'auto-complete))
+
 (eval-after-load 'auto-complete
-  `(ac-define-source shell
+  '(ac-define-source shell
```

Comment:
Semantically the same. I changed it because as a reader, I expect to find a `,` after a ```, because that is the reason to use a backtick instead of a quote.